src/lib/crypto/kimchi_backend/pasta/precomputed.ml

```diff
@@ -6,6 +6,21 @@
     of the curve which is set here 
     https://github.com/MinaProtocol/mina/blob/3d16db598630d8865c04b189b9d41f24bbc73b18/src/lib/zexe_backend/pasta/basic.ml#L17
 *)
+
```

Comment:
IMO this file should not exist. It looks like we can generate it at build time from https://github.com/MinaProtocol/mina/blob/b137fbd750d9de1b5dfe009c12de134de0eb7200/src/lib/pickles/precomputed/gen_values/gen_values.ml

Reply:
I added it because generating it at build time is very slow and adds unnecessary time to the build

Reply:
at least the command to obtain that file should be written somewhere so I can easily compare or update it :o

Reply:
The command should be added to the `dune` file as a 'promote' rule.